projects/core/src/lib/types/keyboard-keys.ts

```diff
@@ -0,0 +1,7 @@
+export const enum KeyboardKeys {
```

Comment:
But now we have a new object in bundle

<img width="787" alt="image" src="https://user-images.githubusercontent.com/12021443/224107388-eca4b4c6-5549-4316-8906-9f210fb996c9.png">


because we use `preserveConstEnums` for compile libs

https://www.typescriptlang.org/play?preserveConstEnums=true#code/KYDwDg9gTgLgBAYwgOwM72MgrgWzgaWAE8AjCAQygBNCjU4BvAKDlbgCFyEBrVML4HAC8cAOScefAaIA0LNgBFgAG2AxBI0UtXrZ81gFFk6qMLFGTetnABaZ0QC8rbAJr2iegL5A